src/clojure_tictactoe/game/board.clj

```diff
@@ -0,0 +1,17 @@
+(ns clojure-tictactoe.game.board)
+
+(defn space-is-open?
+  [space board]
+  (int? (board space)))
+
+(defn mark-space
+  [space marker board]
+  (when (space-is-open? space board)
+    (assoc board space marker)))
+
+(defn render-empty-board []
+  (into [] (take 9 (range))))
+
+(defn board-full?
+  [board]
+  (not (some #(space-is-open? %  board) (take (count board) (range)))))
```

Comment:
This could be cleaned up a bit using the thread macro. Or some `let` variable assignment. But I'd probably suggest going with the threading macro.

Reply:
✅